src/app/attachment/provider/inbox-linshare-attachment-provider.service.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+angular.module('linagora.esn.unifiedinbox.linshare')
+  .factory('inboxLinshareAttachmentProvider', inboxLinshareAttachmentProvider);
+
+function inboxLinshareAttachmentProvider(
+  _,
+  $q,
+  fileUploadService,
```

Comment:
Explicit dependencies are missing

Reply:
There'll be another PR to handle explicit dependencies & translation issues.